tests/unit/components/plate_based_widgets/mapeditor/PlateMapHover.spec.js

```diff
@@ -0,0 +1,415 @@
+    [".well_21", 21, ".plate_21", "Hover"],
+    [".well_22", 22, ".plate_22", "Hover"],
+    [".well_23", 23, ".plate_23", "Hover"],
+  ])(
+    "Given that no wells are selected, When user Hover on %s, Then then well %s visually gets highlighted with %s effect, upon user focus change it gets un-highlighted",
+    async (string_name_of_button, well_indices, selector_str, event) => {
+      const select = [];
+      for (let i = 0; i < 24; i++) {
+        select.push(false);
+      }
+      const color = new Array(24).fill("#b7b7b7");
+
+      const propsData = {
+        selected: select,
+        platecolor: color,
+      };
+      wrapper = mount(ComponentToTest, {
+        propsData,
+        store,
+        localVue,
+      });
+      for (let count = 0; count < 24; count++) {
+        const well1 = wrapper.find(".plate_" + count);
+        expect(well1.attributes("stroke-width")).toBe("0");
+      }
+
+      const well_enter = wrapper.find(selector_str);
+      await well_enter.trigger("mouseenter");
+      await wrapper.vm.$nextTick(); // wait for update
+      expect(wrapper.find(selector_str).attributes("stroke")).toBe("#ececed");
+      expect(wrapper.find(selector_str).attributes("stroke-width")).toBe("2");
+
+      const well_leave = wrapper.find(selector_str);
+      await well_leave.trigger("mouseleave");
+      await wrapper.vm.$nextTick(); // wait for update
+      expect(wrapper.find(selector_str).attributes("stroke")).toBe("#FFFFFF");
+      expect(wrapper.find(selector_str).attributes("stroke-width")).toBe("0");
+    }
+  );
+  test.each([
+    [
+      "Row A",
+      [0, 4, 8, 12, 16, 20],
+      ".span__platemap-editor-row-index-A > label",
+    ],
+    [
+      "Row B",
+      [1, 5, 9, 13, 17, 21],
+      ".span__platemap-editor-row-index-B > label",
+    ],
+    [
+      "Row C",
+      [2, 6, 10, 14, 18, 22],
+      ".span__platemap-editor-row-index-C > label",
+    ],
+    [
+      "Row D",
+      [3, 7, 11, 15, 19, 23],
+      ".span__platemap-editor-row-index-D > label",
+    ],
+    [
+      "Column 01",
+      [0, 1, 2, 3],
+      ".span__platemap-editor-column-index-one > label",
+    ],
+    [
+      "Column 02",
+      [4, 5, 6, 7],
+      ".span__platemap-editor-column-index-two > label",
+    ],
+    [
+      "Column 03",
+      [8, 9, 10, 11],
+      ".span__platemap-editor-column-index-three > label",
+    ],
+    [
+      "Column 04",
+      [12, 13, 14, 15],
+      ".span__platemap-editor-column-index-four > label",
+    ],
+    [
+      "Column 05",
+      [16, 17, 18, 19],
+      ".span__platemap-editor-column-index-five > label",
+    ],
+    [
+      "Column 06",
+      [20, 21, 22, 23],
+      ".span__platemap-editor-column-index-six > label",
+    ],
+  ])(
+    "Given that no wells are selected, When user Hover on %s, Then then wells %s visually become selected (have the stroke outline), user does a hover on rows and columns (have the hover stroke outline)",
```

Comment:
I don't understand what the last part of your test title means. "user does a hover on rows and columns (have the hover stroke outline)".  Was that leftover from an earlier version you had of the test title? or is it still relevant?